### PR TITLE
styling angular material radio and checkboxes

### PR DESCRIPTION
**Pull request by pflopez**
Styles material radio and checkbox inputs.

Screen Shot 2024-06-17 at 15 07 58

### CODE REVIEW

File: src/interface/src/styleguide/form.stories.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    }
+    .flex {
+      display: flex;
+      gap: 16px;
+      flex-direction: column;
+      align-items: flex-start;
+      min-width: 200px;
+    }
+  `,
+  template: `
```
Comment (lastminutediorama) [inline src/interface/src/styleguide/form.stories.ts:43]: is it possible to store this template as a separate file for maintainability, especially if there are other things we may want to add in the future?

File: src/interface/src/styles/_theme.scss [hunk 2]
```diff
@@ -51,11 +52,14 @@ $planscape-typography: mat.define-typography-config(
...
 
+// Create a custom palette using the primary color
+$custom-primary-palette: map.merge(mat.$indigo-palette, (500: $color-main-blue));
 // Define the palettes for your theme using the Material Design palettes available in palette.scss
 // (imported above). For each palette, you can optionally specify a default, lighter, and darker
 // hue. Available color palettes: https://material.io/design/color/
-$planscape-frontend-primary: mat.define-palette(mat.$indigo-palette);
+$planscape-frontend-primary: mat.define-palette($custom-primary-palette);
 
+$custom-error-pallete: map.merge(mat.$red-palette, (500: $color-error));
```
Comment (lastminutediorama) [inline src/interface/src/styles/_theme.scss:62]: I think this should be `palette` not `pallete`

File: src/interface/src/styleguide/form.stories.ts (hunk 1 above)
Comment (pflopez) [reply src/interface/src/styleguide/form.stories.ts:43]: sure thing!

**Review by lastminutediorama — APPROVED** [review]
looks great, i'm glad we have this now

**Pull request merged**